humility-core/src/hubris.rs

```diff
@@ -490,13 +542,14 @@ macro_rules! dump {
                 // cast note type down to u32 since we know the OXIDE_NTs are u32 constants
                 match (note.n_type as u32) {
                     OXIDE_NT_HUBRIS_REGISTERS => {
-                        let mut bytes = [0x0u8; 8];
+                        // 4 bytes are used for register id, the rest are for the register value
+                        let mut bytes = [0x0u8; (4 + size_of::<$abi_size>() as $note_hdr_field_size) as usize];
```

Comment:
Can we add a helper for doing this calcuation? It can be used by this function and above where `TOTAL_ENTRY_SIZE` const is defined.

Reply:
Added the `register_entry_size` macro to help with this.